front end sending properties not in the current pipeline

**Describe the bug**
When switching into driver mode, the front end still sends a lot of properties which driver mode does not have.

`solvePNPEnabled` for example

```
[2024-07-25 14:06:44] [VisionModule - VisionModuleChangeSubscriber - USB_Camera] [ERROR] Could not set prop solvePNPEnabled with value false on CVPipelineSettings{pipelineIndex=-1, pipelineType=DriverMode, inputImageRotationMode=DEG_0, pipelineNickname='Driver Mode', cameraExposureRaw=20.0, cameraBrightness=50, cameraGain=-1, cameraRedGain=-1, cameraBlueGain=-1, cameraVideoModeIndex=4, streamingFrameDivisor=NONE, ledMode=false, inputShouldShow=true, outputShouldShow=true}: solvePNPEnabled
[2024-07-25 14:06:44] [VisionModule - VisionModuleChangeSubscriber - USB_Camera] [ERROR] java.lang.NoSuchFieldException: solvePNPEnabled
        at java.base/java.lang.Class.getField(Class.java:2117)
        at org.photonvision.vision.processes.VisionModuleChangeSubscriber.onDataChangeEvent(VisionModuleChangeSubscriber.java:205)
        at org.photonvision.common.dataflow.DataChangeService.dispatchFromQueue(DataChangeService.java:65)
        at java.base/java.lang.Thread.run(Thread.java:840)
```

**To Reproduce**
Steps to reproduce the behavior:
go to driver mode, look at logs

**Screenshots / Videos**
If applicable, add screenshots to help explain your problem. Additionally, provide journalctl logs and settings zip export. If your issue is regarding the web dashboard, please provide screenshots and the output of the browser console.

**Platform:**
 - reproduced on pi

**Additional context**
no

--- a/photon-client/src/components/dashboard/tabs/InputTab.vue
+++ b/photon-client/src/components/dashboard/tabs/InputTab.vue
@@ -40,7 +40,7 @@ const handleResolutionChange = (value: number) => {
   useCameraSettingsStore().changeCurrentPipelineSetting({ streamingFrameDivisor: getNumberOfSkippedDivisors() }, false);
   useCameraSettingsStore().currentPipelineSettings.streamingFrameDivisor = 0;
 
-  if (!useCameraSettingsStore().isCurrentVideoFormatCalibrated) {
+  if (!useCameraSettingsStore().isCurrentVideoFormatCalibrated && !useCameraSettingsStore().isDriverMode) {
     useCameraSettingsStore().changeCurrentPipelineSetting({ solvePNPEnabled: false }, true);
   }
 };

